src/ort_genai_c.h

```diff
@@ -287,6 +288,8 @@ OGA_EXPORT bool OGA_API_CALL OgaGenerator_IsDone(const OgaGenerator* generator);
 OGA_EXPORT OgaResult* OGA_API_CALL OgaGenerator_ComputeLogits(OgaGenerator* generator);
 OGA_EXPORT OgaResult* OGA_API_CALL OgaGenerator_GenerateNextToken(OgaGenerator* generator);
 
+OGA_EXPORT OgaResult* OGA_API_CALL OgaGenerator_SetRuntimeOptionsConfig(OgaGenerator* generator, const char* key, const char* value);
```

Comment:
@RyanUnderhill - since you are reworking the APIs, if you have a better name, that's fine too.

Reply:
> @RyanUnderhill - since you are reworking the APIs, if you have a better name, that's fine too.

I'm kinda in the middle of both of you, I prefer: `OgaGenerator_SetRuntimeOption`

Reply:
Thanks to both of you, going ahead with Ryan's suggestion.

Reply:
Now that we are deciding on the name, is there any use for the `_` between `OgaGenerator` and the `SetRuntimeOption`? Shall we rename our API to avoid using `_` in the function names?